autupdate.sh

```diff
@@ -27,7 +27,12 @@ while IFS='/' read -r org repo || [[ -n "$repo" ]]; do
   cd $repo
   git fetch origin
   git checkout -B update-dependencies
-  git reset --hard origin/master
+  # Until we switch all default branches to main, flag H2 as an exception
```

Comment:
I wonder if we can get away without knowing the branch name at all?

Reply:
@cbeer Love it. Just tested and it works like a charm.